src/modules/commander/mag_calibration.cpp

```diff
@@ -150,42 +151,42 @@ int do_mag_calibration(orb_advert_t *mavlink_log_pub)
 
 #else
 		(void)sprintf(str, "CAL_MAG%u_XOFF", cur_mag);
-		result = param_set(param_find(str), &mscale_null.x_offset);
+		result = param_set_no_notification(param_find(str), &mscale_null.x_offset);
```

Comment:
This looks problematic. If there is no change notification then the driver side might not pick it up. I believe this should be reverted.